hw/dv/tools/dvsim/testplans/racl_testplan.hjson

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: racl_rw
+      desc: '''
+          Precondition:
+            - Randomize the number of RACL policies and randomize its default policy values.
+            - Randomize the policy selection vector of the CSRs.
+            - Randomize the ErrorRsp parameter.
```

Comment:
This is a parameter of the DUT. It may be possible to randomize this in the DV env, but I'm not certain

Reply:
As with the number of vectors, I think we can just drop this. (If we want to talk about the parameter, I think we're missing a "Racl" prefix)

Reply:
What does removing the parameter mean? That would be to different configurations, right?